homework/hw8/task2_moresad.py

```diff
@@ -0,0 +1,56 @@
+"""Task 2."""
+import sqlite3
+
+
+class TableData:
+    """Class to represent single table data from sqlite."""
+
+    def __init__(self, database_name: str, table_name: str):
+        self.table_name = table_name
+        self.database_name = database_name
+
+    def __len__(self) -> int:
+        """Amount of rows in given table."""
+        self._cursor.execute(f"select count(*) from {self.table_name}")
+        return self._cursor.fetchone()[0]
+
+    def __getitem__(self, item: str):
+        """Get item protocol.
+
+        Get us access to single data row for president with given name
+        Example: presidents['Yeltsin'] # ('Yeltsin', 999, 'Russia')
+        """
+        self._cursor.execute(
+            f"SELECT * from {self.table_name} where name=:name", {"name": item}
```

Comment:
We can place command value to another variable here and below to delete nestation levels